proto/iotics/api/common.proto

```diff
@@ -165,28 +165,13 @@ message SubscriptionHeaders {
 
 }
 
-// HostID is a unique host identifier.
-message HostID {
-  // Host Identifier string representation
-  string value = 1;
-}
-
-// TwinID is a unique twin identifier.
-message TwinID {
+// Twin is the virtual representation of a (physical, purely virtual or hybrid) device,
+// is only ever located in the host it was created.
+message Twin {
```

Comment:
I'd leave this as TwinID since the object is an ID.
I take the point that your access pattern will be twinID.id - maybe then the identifiers can be renamed to did and hostDid

but do we really need the TwinID object and force the clients to having to build it? Or can we not have the two properties (twinID and hostTwinID inlined in the parent object?

I know it's not "pretty" but it'll make the api more usable

Reply:
I get your point about naming. 
The TwinID object is intended to be returned and then forwarded to subsequent requests.
It will be returned for local requests too.
So there is no real benefit here of inlining them in the parent I don't think